package.json

```diff
@@ -37,6 +37,7 @@
     "css-loader": "^0.28.11",
     "exports-loader": "^0.6.2",
     "file-loader": "^1.1.11",
+    "froala-editor": "3.1.1",
```

Comment:
why to add in dev dependency?

Reply:
The `npm run demo` script still requires the `froala-editor` dependency to be present, so it must be declared as a dev dependency.

May I ask why this pull request was closed?